tests/unit/all.js

```diff
@@ -0,0 +1,10 @@
+define([
+	"tests/unit/accordion/core",
+	"tests/unit/accordion/events",
+	"tests/unit/accordion/methods",
+	"tests/unit/accordion/options",
+	"tests/unit/autocomplete/core",
+	"tests/unit/autocomplete/events",
+	"tests/unit/autocomplete/methods",
+	"tests/unit/autocomplete/options",
```

Comment:
We should create an `all` module for each widget, or perhaps just `tests/unit/accordion.js` as the module.